### PR TITLE
[store] Update AppStore translations

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org) for [OrganicMaps/Apple AppStore Metadata](https://hosted.weblate.org/projects/organicmaps/apple-appstore-metadata/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: iphone/metadata/ru/name.txt [hunk 1]
```diff
@@ -1 +1 @@
-Organic Maps офлайн карты
+Organic Maps хайкинг офлайн
```
Comment (rtsisyk) [inline iphone/metadata/ru/name.txt:1]: Наверное, лучше offline оставить?
@organicmaps/translations-ru

File: iphone/metadata/ru/promotional_text.txt [hunk 2]
```diff
@@ -1 +1 @@
-Пожалуйста, исправляйте все проблемы с картой на OpenStreetMap.org, и сообщайте нам о любых ошибках. Наша небольшая команда будет рада вам помочь. Удачных путешествий!
+Исправляйте, пожалуйста, все ошибки на карте на OpenStreetMap.org, а нам сообщайте о багах. Улучшать приложение нашу небольшую команду мотивирует обратная связь от вас.
```
Comment (rtsisyk) [inline iphone/metadata/ru/promotional_text.txt]: Google Translate?

File: iphone/metadata/en-US/subtitle.txt [hunk 3]
```diff
@@ -1 +1 @@
-Trails, Travel, GPS Navigation
+Navigate with Privacy
```
Comment (rtsisyk) [inline iphone/metadata/en-US/subtitle.txt:1]: @oleg-pah I have doubts that removal of important keywords here will help with ASO, but let's try once.

File: iphone/metadata/de-DE/name.txt [hunk 4]
```diff
@@ -1 +1 @@
-Organic Maps Offline Karten
+Organic Maps: Wandern Fahren
```
Comment (rtsisyk) [inline iphone/metadata/de-DE/name.txt:1]: Removes important keywords, but let's try and see.

**Review by rtsisyk — APPROVED** [review]

File: iphone/metadata/de-DE/description.txt [hunk 5]
```diff
@@ -1,39 +1,10 @@
-• Detaillierte Offline-Karten mit Orten, die es auf anderen Karten nicht gibt
-• Rad-, Wander- und Spazierwege
-• Höhenlinien, Höhenprofile, Bergspitzen und Steigungen
-• Schritt-für-Schritt-Navigation für Fußgänger, Radfahrer und Autos mit Sprachführung
-• Schnelle Offlinesuche
-• Export und Import von Lesezeichen als KML, KMZ, GPX
```
Comment (biodranik) [inline iphone/metadata/de-DE/description.txt:6]: Important keywords were lost

File: iphone/metadata/de-DE/description.txt [hunk 6]
```diff
@@ -1,39 +1,10 @@
-• Detaillierte Offline-Karten mit Orten, die es auf anderen Karten nicht gibt
-• Rad-, Wander- und Spazierwege
-• Höhenlinien, Höhenprofile, Bergspitzen und Steigungen
```
Comment (biodranik) [inline iphone/metadata/de-DE/description.txt:3]: Important keywords were lost.

File: iphone/metadata/de-DE/subtitle.txt [hunk 7]
```diff
@@ -1 +1 @@
-GPS Navigation, Wandern
```
Comment (biodranik) [inline iphone/metadata/de-DE/subtitle.txt:1]: Important keywords were lost. They were not duplicated in keywords list. This change filtered out all searches of "GPS maps" or "GPS navigation" in the store.

**Pull request merged**